saos-search/src/test/java/pl/edu/icm/saos/search/search/service/JudgmentSearchServiceTest.java

```diff
@@ -493,7 +493,7 @@ private SolrInputDocument fetchScJudgmentDoc() {
         
         doc.addField("judgmentType", "RESOLUTION");
         
-        doc.addField("scJudgmentForm", "wyrok SN");
+        doc.addField("scJudgmentFormId", 12l);
```

Comment:
I would add here some value to `scJudgmentFormName` and check if that value is equals to `result.getScJudgmentFormName()` in `search_CHECK_SUPREME_COURT_RESULT` test